crates/sdk/macros/src/state.rs

```diff
@@ -50,6 +50,12 @@ impl ToTokens for StateImpl<'_> {
             impl #impl_generics ::calimero_sdk::state::AppState for #ident #ty_generics #where_clause {
                 type Event<#lifetime> = #event;
             }
+
+            impl #impl_generics #ident #ty_generics #where_clause {
+                fn external() -> ::calimero_sdk::env::ext::External {
+                    ::calimero_sdk::env::ext::External {}
```

Comment:
while this works, `External` is defined without them, so maybe this?

```suggestion
                    ::calimero_sdk::env::ext::External
```